GenerateJSCode.php

```diff
@@ -230,12 +230,14 @@ public function generateInitialJSCode($datasource, $options, $dbspecification, $
 
         $this->generateAssignJS(
             "INTERMediatorOnPage.isNativeAuth",
-            (isset($options['authentication']) && isset($options['authentication']['user'])
-                && ($options['authentication']['user'][0] === 'database_native')) ? "true" : "false");
+            ((isset($options['authentication'])
+                && isset($options['authentication']['user'])
+                && ($options['authentication']['user'][0] === 'database_native'))
+                || ((new LDAPAuth())->isActive)) ? "true" : "false");
```

Comment:
IM's class autoloader depends on the spl_autoload_register function. It's supported from 5.1.2. Doesn't it exist the SPL function support?